backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/view/ViewPropertiesConfigurer.java

```diff
@@ -278,4 +306,48 @@ private SelectDescription createShapeSelectionField() {
         };
     }
 
+    private Function<VariableManager, List<Object>> getDiagnosticsProvider(Object feature) {
+        return variableManager -> {
+            var optionalSelf = variableManager.get(VariableManager.SELF, EObject.class);
+
+            if (optionalSelf.isPresent()) {
+                EObject self = optionalSelf.get();
+                List<Object> validate = this.validationService.validate(self, feature);
```

Comment:
validate should be renamed to diagnostics since this is what the variable contains